modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -349,4 +349,89 @@ The output file is:
 k/J1EHyxJagOcyQELoaqIDCzzKc1bLGJiHY2dv6jzuhtVpP1LTlgK3y8EH7OkK5iuuu8kik3P5dCDMixZXwn/EV6xbUwLFKnEb4x1pUX6aopLP/KULm0pQ==
 ----
 
-Note that key sizes might depend on the Java Cryptography Extension (JCE) version. For example, since version 1.8.0_161, the AES key size needs to be of at least 32 characters (256 bytes), instead of 16 for previous versions.
+=== Arguments Reference
+
+[cols="20a,50a,30a", options="header"]
+|===
+|Argument
+|Description
+|Accepted Values
+
+|`string`
+|Sets decryption mode to `string`. +
+Encrypts a single string value provided as argument `<value>`.
+|
+* `string`
+* `file`
+* `file-level`
+
+Only one of these can be set per operation.
+
+|`file`
+|Sets decryption mode to `file`. +
+Encrypts the value of each property set in the properties file provided as argument `<input file>`.
+|
+* `string`
+* `file`
+* `file-level`
+
+Only one of these can be set per operation.
+
+|`file-level`
+|Sets decryption mode to `file-level`. +
+Encrypts the whole content of the properties file provided as argument `<input file>`.
+|
+* `string`
+* `file`
+* `file-level`
+
+Only one of these can be set per operation.
+
+|`encrypt`
```

Comment:
See above.
I have no idea what we could put in the third column for these. Encrypt and decrypt are two separate arguments with only one value apiece....